pisces/config.py.deploy

```diff
@@ -41,3 +41,4 @@ TERM_SCHEMA = "${TERM_SCHEMA}"
 NOTIFY_EMAIL = ${NOTIFY_EMAIL}
 NOTIFY_TEAMS = ${NOTIFY_TEAMS}
 TEAMS_URL = "${TEAMS_URL}"
+EXTENT_NULLABLE_FLG = ${EXTENT_NULLABLE_FLG}
```

Comment:
I'm wondering if we can call this variable `INHERIT_EXTENT` because that's really what the application logic is doing. Also, that way if we want to make other inheritance rules configurable all those configs will end up next to each other in an alphabetical list. Small thing, but does make a difference when there's a lot of configs!